code/modules/surgery/generic.dm

```diff
@@ -303,7 +303,7 @@
 /datum/surgery_step/generic/amputate
 	allowed_tools = list(
 	/obj/item/weapon/surgical/circular_saw = 100, \
-	/obj/item/weapon/material/hatchet = 75
+	/obj/item/weapon/material/knife/machete/hatchet = 75
```

Comment:
allow non-hatchet machetes to be used as csaws?

Reply:
Nah, hatchet makes sense, IMO, regular machete does not.